src/action_event.rs

```diff
@@ -44,29 +44,29 @@ impl Kurinji {
         let mut _actions = input_map.action_raw_strength.clone();
         for (a, s) in input_map.action_prev_strength.iter() {
             if !_actions.contains_key(a) {
-                _actions.insert(a.clone(), s.clone());
+                _actions.insert(a.clone(), *s);
```

Comment:
It is better to use an explicit copy/clone call. It is clearer than deref "*" since in cases where types don't implement copy it will result in a move